scipy/ndimage/filters.py

```diff
@@ -643,9 +643,45 @@ def correlate(input, weights, output=None, mode='reflect', cval=0.0,
     %(cval)s
     %(origin_multiple)s
 
+    Returns
+    -------
+    result : ndarray
+        The result of correlation of `input` with `weights`.
+
     See Also
     --------
     convolve : Convolve an image with a kernel.
+
+    Examples
+    --------
+    Correlation is the process of moving a filter mask often referred to
+    as kernel over the image and computing the sum of products at each location.
+    >>> from scipy.ndimage import correlate
+    >>> input_img = np.arange(25).reshape(5,5)
+    >>> print(input_img)
+    [[ 0  1  2  3  4]
+    [ 5  6  7  8  9]
+    [10 11 12 13 14]
+    [15 16 17 18 19]
+    [20 21 22 23 24]]
+
+    Define a kernel (weights) for correlation. In this example, it is for sum of
+    center and up, down, left and right next elements.
+
+    >>> weights = [[0, 1, 0],
+    ...            [1, 1, 1],
+    ...            [0, 1, 0]]
+
+    We can calculate a correlation result:
+    For example, element [2,2] is 7 + 11 + 12 + 13 + 17 = 60
```

Comment:
Could you use double backticks around `[2,2]` and the `7 + ...` expression? Will look nicer in the rendered html.